include/Dialect/LWE/IR/LWETypes.td

```diff
@@ -0,0 +1,58 @@
+#ifndef INCLUDE_DIALECT_LWE_IR_LWETYPES_TD_
+#define INCLUDE_DIALECT_LWE_IR_LWETYPES_TD_
+
+include "include/Dialect/LWE/IR/LWEDialect.td"
+include "include/Dialect/LWE/IR/LWEAttributes.td"
+
+include "include/Dialect/Polynomial/IR/PolynomialAttributes.td"
+
+include "mlir/IR/DialectBase.td"
+include "mlir/IR/AttrTypeBase.td"
+
+// A base class for all types in this dialect
+class LWE_Type<string name, string typeMnemonic>
+    : TypeDef<LWE_Dialect, name> {
+  let mnemonic = typeMnemonic;
+}
+
+// LWE Ciphertexts are ranked tensors of integers representing the LWE samples
+// and the bias.
+def LWECiphertext : LWE_Type<"LWECiphertext", "lwe_ciphertext"> {
+  let summary = "A type for an LWE ciphertexts";
```

Comment:
`A type for an LWE ciphertexts` -> `A type for LWE ciphertexts` or `A type for an LWE ciphertext`

Reply:
Done